instrumentation/jmx-metrics/javaagent/src/main/resources/jmx/rules/tomcat.yaml

```diff
@@ -4,45 +4,9 @@
 # would match too broadly
```

Comment:
Replace "thus we have to duplicate metrics definitions as using a wildcard would match too broadly" by "thus we use both MBean names for most of the metrics"